common/network-common/src/main/java/org/apache/spark/network/util/DBProvider.java

```diff
@@ -38,17 +38,18 @@ public static DB initDB(
         StoreVersion version,
         ObjectMapper mapper) throws IOException {
       if (dbFile != null) {
-        switch (dbBackend) {
-          case LEVELDB:
+        return switch (dbBackend) {
+          case LEVELDB -> {
             org.iq80.leveldb.DB levelDB = LevelDBProvider.initLevelDB(dbFile, version, mapper);
             logger.warn("The LEVELDB is deprecated. Please use ROCKSDB instead.");
-            return levelDB != null ? new LevelDB(levelDB) : null;
-          case ROCKSDB:
+            yield levelDB != null ? new LevelDB(levelDB) : null;
+          }
+          case ROCKSDB -> {
             org.rocksdb.RocksDB rocksDB = RocksDBProvider.initRockDB(dbFile, version, mapper);
-            return rocksDB != null ? new RocksDB(rocksDB) : null;
-          default:
-            throw new IllegalArgumentException("Unsupported DBBackend: " + dbBackend);
-        }
+            yield rocksDB != null ? new RocksDB(rocksDB) : null;
+          }
+          default -> throw new IllegalArgumentException("Unsupported DBBackend: " + dbBackend);
```

Comment:
How about

```scala
return switch (dbBackend) {
          case LEVELDB -> {
            logger.warn("The LEVELDB is deprecated. Please use ROCKSDB instead.");
            yield new LevelDB(LevelDBProvider.initLevelDB(dbFile, version, mapper));
          }
          case ROCKSDB -> new RocksDB(RocksDBProvider.initRockDB(dbFile, version, mapper));
        };
```
`dbBackend` is an enumeration type, the default case is not necessary.

Reply:
Please ignore this comments

Reply:
We can remove the `default` here.

Reply:
OK ~